guide/C/ch_basics.xml

```diff
@@ -603,7 +603,7 @@ Translators:
             the process, back up to change something, or apply to confirm your choices. If you're
             creating a new book, &app; will present a dialog to select a backend and either a file
             to save it or database server, database name, and credentials to create a MySQL or
-            Postgresql database if GnuCash was built with those features and you have the necessary
+            PostgreSQL database if GnuCash was built with those features and you have the necessary
```

Comment:
I wonder if the 3 SQL apps should be added to the DTD with markup.

Reply:
That is a difficult problem.
For example, the official name of the application is MySQL.
However, in the select box it is mysql. The same is true for PostgreSQL, SQLite3 and XML.
If the writer and translator do not misunderstand it, you can add it as ENTITY.

Currently, my priority is to fix typo that interferes with translation.

Reply:
That can be resolved by defining app-foo <application>Foo</application> and
gnc-foo <userinput>foo</userinput>.